### ISSUE TITLE
just wonder when see first demo building or at least just video update

### ISSUE THREAD

**Issue by Yoshi390** [Trixarian/EtherealSX2#5]
Understand that App bring done with free time just make get worry no visual update all wonder like see demo apk at all or at least some video don't if realistic for us see ver of great app sure to be. 

Like not try see entities just useful not good sign don't see update for app development in mouths and sign because alot trouble could have more one person on app help it along. Or just going wait Patrice on it.

**Comment by Yoshi390** [comment]
Yes am happy work you did with NetherSX2 Patch and goal of port modern pcsx2 to android itself. Just wonder if see demo ver of app soon so see where it is at and outhers people help along with it so bring back good ps2 emulation to android.

**Comment by Trixarian** [comment]
That's not a good idea at the moment consider what I have is barely functional, and even MYPS2 functions better than it does. People are expecting AetherSX2 2.0 off the bat and it's just not even close to that degree of quality

I admit this is partially self-inflicted with the hype I created when I was excited about finally having a name for it. When I started I had no experience creating an Android App, much less an emulator, and the closest thing I've created in the same line was a terminal emulator (for playing MUDs) and writing my own games. Other than having some knowledge of translation layers from writing them for server protocols and romhacking, this is a new experience that requires me to learn a lot of new things - which with my time constraints, puts even more pressure on what time I can dedicate to everything. To be completely honest, it's been one of the largest contributing factors to why I've had so much trouble balancing everything, and why the final release of NetherSX2 has been taking so goddamn long - Ethereal keeps striking again and drawing my attention away with something new breaking on me. It's both interesting and frustrating at the same time

But I digress. The TL;DR of the situation is that what I have isn't even pre-alpha level and I feel it would be a terrible representation of what Ethereal can be, especially with the crowd expecting something better than AetherSX2, and I'd rather wait until I have something to show that won't disappoint those on the hype train. Either way it won't be AetherSX2 2.0 - it may get there eventually - but at the start it'll be dead simple and based on automation, with more configuration options coming as I work on it

**Comment by Ottorock** [comment]
Yoshi390 it’ll come just hang in their please and thank you

**Comment by Kenny417** [comment]
Thanks for all the work you're doing on this. I think you're going to start getting flooded with similar comments a lot in 2025 since RetroAchievements is dropping support for Hardcore mode on NetherSX2 and there are no other viable options on Android for PS2. 

Are you really the only person with enough coding experience who can build this emulator? Or are there other devs who could jump into the trenches with you to help with this? Open source would definitely be the way to go with this so we don't end up in another AetherSX2 situation in the future. (Although I'm sure you know this with all the work you've had to do with NetherSX2)

Also, I will gladly pay for a "Gold" version if you follow a PPSSPP-like model with the app.

**Comment by Ottorock** [comment]
Kenny147 I agree with you a gold version of EtherealSX2 tho it needs other colors as a side dish like silver bronze crystal asteroid cosmos space any color would work to

**Comment by Yoshi390** [comment]
Yes agree with this true just wait for app come out and some time before a good port of pcsx2 android can't wait day have playable ps2 emulator with good update base off of pcsx2

**Comment by hb-robo** [comment]
> Are you really the only person with enough coding experience who can build this emulator? Or are there other devs who could jump into the trenches with you to help with this? Open source would definitely be the way to go with this so we don't end up in another AetherSX2 situation in the future. (Although I'm sure you know this with all the work you've had to do with NetherSX2)

Strongly agree, this project is quite the undertaking for any one person. PCSX2 parity is a wonderful goal but that team has the benefit of a much larger open-source team with much longer tails in the project, so to speak. To me (the peanut gallery), it follows that a similar model is also in the best interest for EtherealSX2.

And yeah, @Trixarian hopefully the RetroAchievements situation doesn't put too much of a burden on you. There are much bigger fish to fry before dealing with that.

**Comment by Trixarian** [comment]
I'll likely use a PR based commit system similar to what PCSX2 uses. I'm not great at doing changelogs since I do bulk changes, and when one change is in 2/3 of the PS2 library, that list of changes can get very long (considering there's over 4k games in the PS2 library). It will be completely open source once I have something decent to share that isn't just a simplified PS2 launcher with a very unstable core, and I don't really care what people do with it. If someone wants to make a RetroArch core from it, then all the more power to them ^^

**Comment by Stormm232** [comment]
An actual god sent.

**Comment by mhamz-iit** [comment]
Happy to contribute $$$ to this incredible project if you set up something.
